packages/myst-to-typst/src/index.ts

```diff
@@ -454,6 +454,27 @@ const handlers: Record<string, Handler> = {
     state.write(']');
     state.ensureNewLine();
   },
+  card(node, state) {
+    if (node.url) {
+      node.children?.push({ type: 'paragraph', children: [{ type: 'text', value: node.url }] });
+    }
+    state.renderChildren(node);
+    state.ensureNewLine();
+    state.write('\n');
+  },
+  cardTitle(node, state) {
+    state.write('*');
```

Comment:
These are now just bold, rather than headings. That means they will not show up in the toc, but they may now be dangling at the bottom of a page in the final PDF. Not really a big deal. Probably need to give cards more thought if they become widespread in typst exports.